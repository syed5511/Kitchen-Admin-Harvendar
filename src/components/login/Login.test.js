import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Login from './Login.component';

describe('Login', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Login />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    shallow(<Login />);
  });
});
