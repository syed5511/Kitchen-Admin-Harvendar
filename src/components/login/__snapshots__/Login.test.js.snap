// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders correctly 1`] = `
<div
  class="login-container"
>
  <div
    class="img-container"
  />
  <div
    class="inner-container"
  >
    <div
      class="login"
    >
      <h3>
        Kitchen
      </h3>
      <p>
        Welcome Back, Please login to your account
      </p>
      <form
        action="#"
        class="login-form"
        className="av-valid"
        method="get"
        noValidate={true}
        onKeyDown={[Function]}
        onSubmit={[Function]}
      >
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="username"
          >
            Username
          </label>
          <input
            className="is-untouched is-pristine av-valid form-control"
            id="username"
            maxLength={24}
            minLength={6}
            name="username"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onInput={[Function]}
            onKeyUp={[Function]}
            required={true}
            type="text"
            value=""
          />
        </div>
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="password"
          >
            Password
          </label>
          <input
            className="is-untouched is-pristine av-valid form-control"
            id="password"
            minLength={8}
            name="password"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onInput={[Function]}
            onKeyUp={[Function]}
            required={true}
            type="password"
            value=""
          />
        </div>
        <div
          class="horizontal-container"
        >
          <div
            className="form-check"
          >
            <label
              className="form-check-label"
            >
              <input
                className="form-check-input"
                type="checkbox"
              />
              Remember me
            </label>
          </div>
          <label
            className=""
          >
            Forgot Password
          </label>
        </div>
        <div
          class="horizontal-container"
        >
          <button
            aria-label={null}
            className="btn btn-primary"
            onClick={[Function]}
          >
            Login
          </button>
          <button
            aria-label={null}
            className="btn btn-secondary"
            onClick={[Function]}
          >
            Sign Up
          </button>
        </div>
      </form>
      <p>
        Term of use. Privacy policy
      </p>
    </div>
  </div>
</div>
`;
